native/extensions/assets-manager/AssetsManagerEx.h

```diff
@@ -58,6 +58,7 @@ class CC_EX_DLL AssetsManagerEx : public RefCounted {
         DOWNLOADING_MANIFEST,
         MANIFEST_LOADED,
         NEED_UPDATE,
+        PARPER_UPDATING,
```

Comment:
What does this event mean?

Reply:
Indicates that the prepare update function is being processed

Reply:
Is `PARPER` a typo?

Reply:
Yes, fiexed.